js-miniapp-sample/package.json

```diff
@@ -17,6 +17,7 @@
     "redux-thunk": "^2.3.0"
   },
   "scripts": {
+    "format": "prettier --write 'src/**/*.js'",
```

Comment:
This is useful 👍  Do you need to also add `--config ./prettier.config.js` to this command?